Cargo.toml

```
@@ -17,5 +17,5 @@ edition = "2021"
 
 [workspace.dependencies]
 diff = { version = "0.1.12", default_features = false }
-regex = { version = "1", default_features = false }
-regex-syntax = { version = "0.6", default_features = false }
+regex = { version = "1.8", default_features = false, features = ["std"] }
```

```suggestion
regex = { version = "1", default_features = false, features = ["std"] }
```
If `1` is possible instead of `1.8`, the minor version seems to go with just `1`. We will be able to mark it `1.8` for lalrpop 1.0

---

I guess? I'm a bit unclear but if that's needed then I'm not opposed to it.

In some sense, isn't part of the issues lalrpop had with regex is because the version was only declared as `version = "1"`? #750 Also the edition was updated recently which seems like a similar kind of breaking change?

---

Seems like we are airing on the side of being conservative here which is fine.

There is a pedantic point to be made here which is that lalrpop doesn't actually compile with regex =1.0 because it uses the `std` feature that landed in 1.3(replacing a now deprecated `use_std` https://github.com/rust-lang/regex/blob/master/CHANGELOG.md#130-2019-09-03). I'm not sure if there is an automatic way to check this, but its something to be careful about.